assets/admin/js/globalpayments-modal.js

```diff
@@ -0,0 +1,70 @@
+/*global jQuery */
+( function( $ ) {
+	'use strict';
+
+	/**
+	 * WooCommerce Backbone Global Payments Modal plugin
+	 *
+	 * @param {object} options
+	 */
+	$.fn.WCGlobalPaymentsPayOrderBackboneModal = function( options ) {
+		return this.each( function() {
+			( new $.WCGlobalPaymentsPayOrderBackboneModal( $( this ), options ) );
+		});
+	};
+
+	/**
+	 * Initialize the Backbone Modal
+	 *
+	 * @param {object} element [description]
+	 * @param {object} options [description]
+	 */
+	$.WCGlobalPaymentsPayOrderBackboneModal = function( element, options ) {
+		var settings = $.extend( {}, $.WCBackboneModal.defaultOptions, options );
+
+		if ( settings.template ) {
+			new $.WCGlobalPaymentsPayOrderBackboneModal.View({
+				target: settings.template,
+				string: settings.variable
+			});
+		}
+	};
+
+	$.WCGlobalPaymentsPayOrderBackboneModal.View = $.WCBackboneModal.View.extend({
+		events: _.extend( $.WCBackboneModal.View.prototype.events, {
+			'click #place_order': 'payOrder'
+		}),
+		payOrder: function( e ){
```

Comment:
We should have a space before the curly brace opening, to keep consistent with the WordPress coding standards.
This applies to all 3 functions in here